File: C/game.c
# include <stdio.h>
# include <stdlib.h>
# include <time.h>

int main()
{   
    int guess, number, count;
    srand(time(0));
    number = rand()%100;
    count = 0;

    while (guess != number)
    {
        printf("Enter Your Guess : ");
        scanf("%d", &guess);

        if (guess < number)
        {
            printf("Increase the Guess !\n");
        }
        else if (guess > number)
        {
            printf("Decrease the Guess !\n");
        }
        count ++;
    }

    printf("You sucessfully guessed the number in %d attempts!", count);
    
    return 0;
}

File: Python/game.py
import random

n = random.randint(0, 100)

action = True

c = 0

while action:
    n1 = int(input("Enter your guess : "))
    
    if n > n1:
        print("More than the current Guess! ")
    
    elif n < n1:
        print("Less than the current Guess! ")

    else:
        action = False
    c += 1

print("You successfully guessed the number in", c, "attempts.")